src/main/java/emissary/core/sentinel/Sentinel.java

```diff
@@ -285,17 +277,22 @@ public void incrementTimer(long time) {
         public void clear() {
             this.agentId = "";
             this.shortName = "";
-            this.placeName = "";
+            this.directoryEntryKey = "";
             resetTimer();
         }
 
+        @Override
+        public int compareTo(Tracker o) {
+            return this.agentName.compareTo(o.agentName);
+        }
+
         @Override
         public String toString() {
-            return new StringJoiner(", ", "[", "]")
-                    .add("agentName='" + agentName + "'")
-                    .add("placeName='" + placeName + "'")
-                    .add("shortName='" + shortName + "'")
-                    .add("timer=" + timer + " minute(s)")
+            return new StringJoiner(", ", "{", "}")
+                    .add("\"agentName\":\"" + agentName + "\"")
+                    .add("\"directoryEntry\":\"" + directoryEntryKey + "\"")
+                    .add("\"shortName\":\"" + shortName + "\"")
+                    .add("\"timeInMinutes\":" + timer)
```

Comment:
confirming you intend this to print like: `"timeInMinutes":2` and not `"timeInMinutes":"2"`